bindings/ergo-lib-wasm/src/address.rs

```diff
@@ -179,3 +209,54 @@ impl From<chain::address::Address> for Address {
         Address(a)
     }
 }
+
+/// Combination of an Address with a network
+/// These two combined together form a base58 encoding
+#[wasm_bindgen]
+#[derive(PartialEq, Eq, Debug, Clone)]
+pub struct NetworkAddress(chain::address::NetworkAddress);
```

Comment:
I created a `NetworkAddress` since it's easier to work with in Yoroi if we can just keep track of which network the address was for everywhere since Yoroi supports both testnet & mainnest all from the same extension.